ethcore/src/miner/miner.rs

```diff
@@ -526,8 +537,8 @@ impl Miner {
 		}
 	}
 
-	fn update_gas_limit(&self, chain: &MiningBlockChainClient) {
-		let gas_limit = chain.best_block_header().gas_limit();
+	fn update_gas_limit(&self, client: &MiningBlockChainClient) {
+		let gas_limit = client.best_block_header().gas_limit();
```

Comment:
it doesn't seem that `MiningBlockChainClient` is necessary here, just `BlockChainClient`.